controls/SV-230253.rb

```diff
@@ -13,40 +13,41 @@
  This setting is not recommended on computers without the hardware random
 generator because insufficient entropy causes the connection to be blocked
 until enough entropy is available.'
-  desc 'check', 'Verify the operating system SSH server uses strong entropy with the
-following command:
+  desc 'check', 'Note: If the operating system is RHEL versions 8.0 or 8.1, this requirement is not applicable.
 
-    Note: If the operating system is RHEL versions 8.0 or 8.1, this requirement
-is not applicable.
+Verify the operating system SSH server uses strong entropy with the following command:
 
-    $ sudo grep -i ssh_use_strong_rng /etc/sysconfig/sshd
+$ sudo grep -i ssh_use_strong_rng /etc/sysconfig/sshd
 
-    SSH_USE_STRONG_RNG=32
+SSH_USE_STRONG_RNG=32
 
-    If the "SSH_USE_STRONG_RNG" line does not equal "32", is commented out
-or missing, this is a finding.'
+If the "SSH_USE_STRONG_RNG" line does not equal "32", is commented out, or is missing, this is a finding.'
   desc 'fix', 'Configure the operating system SSH server to use strong entropy.
 
 Add or modify the following line in the "/etc/sysconfig/sshd" file.
 
 SSH_USE_STRONG_RNG=32
 
-The SSH service must be restarted for changes to take effect.'
+The SSH service must be restarted for changes to take effect.
+
+Restart the SSH Daemon with the following command:
+
+$ sudo systemctl restart sshd.service'
   impact 0.3
   ref 'DPMS Target Red Hat Enterprise Linux 8'
   tag severity: 'low'
   tag gtitle: 'SRG-OS-000480-GPOS-00227'
   tag gid: 'V-230253'
-  tag rid: 'SV-230253r627750_rule'
+  tag rid: 'SV-230253r1044799_rule'
   tag stig_id: 'RHEL-08-010292'
-  tag fix_id: 'F-32897r567506_fix'
+  tag fix_id: 'F-32897r1044798_fix'
   tag cci: ['CCI-000366']
   tag nist: ['CM-6 b']
   tag 'host'
   tag 'container-conditional'
 
-  only_if('Control not applicable - SSH is not installed within containerized RHEL', impact: 0.0) {
-    !(virtualization.system.eql?('docker') && !file('/etc/sysconfig/sshd').exist?)
+  only_if('Control not applicable - The system is RHEL 8/8.1 and/or SSH is not installed within containerized RHEL', impact: 0.0) {
+    !(virtualization.system.eql?('docker') && !file('/etc/sysconfig/sshd').exist? && command("grep -e 8.0 -e 8.1 /etc/redhat-release").stdout.strip)
```

Comment:
We should use the `os` resource for this check. 
The upstream InSpec os resource provides `os.release`, which you can use like this:
`os.release.to_f >= 8.4`
But this has a bug for releases with multi-digit minor numbers which is being handled in this PR:
https://github.com/mitre/redhat-enterprise-linux-8-stig-baseline/pull/39/files

Depending on the order of PRs being merged, we can use `os.release` or `os.version.minor`